client/src/styles/mfa-variables.scss

```diff
@@ -44,20 +49,56 @@ $loading-indicator-animation: spinner-loader $loading-indicator-speed infinite l
 
 // Authentication method tiles
 $tile-border-color: $color-grey-border-light;
+$tile-border-color-dark: $color-grey-border-light;
 $tile-border-color-hover: lighten($tile-border-color, 9.5);
+$tile-border-color-hover-dark: darken($tile-border-color-dark, 40);
 $tile-border-color-active: rgb(0, 111, 199);
+$tile-border-color-active-dark: var(--color-green);
```

Comment:
This should probably be a lighter shade of blue, rather than a totally different colour?